src/yatm_v2/src/app/cli.rs

```diff
@@ -488,19 +496,77 @@ pub async fn cli() -> Result<()> {
                     "Connecting to repository: {}/{}",
                     config.repo_owner, config.repo_name
                 );
-                let github_issues = gh.get_issues(Some(State::Open)).await?;
+                let github_issues = gh.get_issues(Some(State::All)).await?;
+
+                let matched_issues = get_local_issues_matches(&local_issues, &github_issues);
+                for i in &matched_issues {
+                    match i.match_type {
+                        IssueMatchType::Match => println!(
+                            "Matched: {} -- {}",
+                            i.local_issue.title,
+                            i.github_issue.as_ref().unwrap().html_url
```

Comment:
It'd be nice to handle the case where the unwrap fails, at least letting the user know what has failed.